### ISSUE TITLE
OpenAPI v3.0 Support

### ISSUE THREAD

**Issue by philsturgeon** [elasticio/odata2openapi#27]
Hey there! Somebody was hunting for an OData to OpenAPI converter on my slack channel and this popped up. They've had to take a multi-step approach of converting Odata to OAS v2.0, then use Apimatic Transformer to get from v2.0 to v3.0, but they got there! :D 

If you plan to keep working on this tool and add v3.0 support, I'd love to get this on [openapi.tools](http://openapi.tools/). Is v3.0 support on the horizon?

**Comment by jhorbulyk** [comment]
It's not clear to me that this library is being actively maintained by the original maintainers.  I've found and made a couple of bugs/improvements that haven't been acknowledged by the original maintainers.  As a result, I have a fork (https://github.com/elasticio/odata2openapi) with my fixes.  I'd be open to such a contribution to my fork.

**Comment by philsturgeon** [comment]
Fair enough. @jhorbulyk could you send a PR to [openapi.tools](https://github.com/apisyouwonthate/openapi.tools/blob/master/_data/tools.yml) with your fork, then if this project shows signs of life and merges stuff, answers, etc, then we can change that in the future. :)

**Comment by akorchev** [comment]
@jhorbulyk unfortunately we can no longer maintain this library due to a shift of work related priorities. I am willing to transfer ownership of this github repo and the NPM to you or anybody willing to take over. Let me know!

**Comment by jhorbulyk** [comment]
@akorchev I'd be willing to accept ownership of this github repo.  Can you contact me at `jacob@elastic.io`?

**Comment by snebjorn** [comment]
Any progress on the OpenAPI v3 support?
